src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithUnionFW.java

```diff
@@ -0,0 +1,235 @@
+    }
+
+    @Override
+    public ListWithUnionFW wrap(
+        DirectBuffer buffer,
+        int offset,
+        int maxLimit)
+    {
+        super.wrap(buffer, offset, maxLimit);
+        final long bitmask = bitmask();
+        int fieldLimit = offset + FIRST_FIELD_OFFSET;
+        for (int field = FIELD_INDEX_UNION_OCTETS; field < FIELD_INDEX_FIELD1 + 1; field++)
+        {
+            switch (field)
+            {
+            case FIELD_INDEX_UNION_OCTETS:
+                if ((bitmask & (1 << FIELD_INDEX_UNION_OCTETS)) != 0)
+                {
+                    unionOctetsRO.wrap(buffer, fieldLimit, maxLimit);
+                    fieldLimit = unionOctetsRO.limit();
+                }
+                break;
+            case FIELD_INDEX_FIELD1:
+                if ((bitmask & (1 << FIELD_INDEX_FIELD1)) != 0)
+                {
+                    optionalOffsets[FIELD_INDEX_FIELD1] = fieldLimit;
+                    fieldLimit += FIELD_SIZE_FIELD1;
+                }
+                break;
+            }
+        }
+        checkLimit(limit(), maxLimit);
+        return this;
+    }
+
+    @Override
+    public ListWithUnionFW tryWrap(
+        DirectBuffer buffer,
+        int offset,
+        int maxLimit)
+    {
+        if (null == super.tryWrap(buffer, offset, maxLimit))
+        {
+            return null;
+        }
+        final long bitmask = bitmask();
+        int fieldLimit = offset + FIRST_FIELD_OFFSET;
+        for (int field = FIELD_INDEX_UNION_OCTETS; field < FIELD_INDEX_FIELD1 + 1; field++)
+        {
+            switch (field)
+            {
+            case FIELD_INDEX_UNION_OCTETS:
+                if ((bitmask & (1 << FIELD_INDEX_UNION_OCTETS)) != 0)
+                {
+                    if (null == unionOctetsRO.tryWrap(buffer, fieldLimit, maxLimit))
+                    {
+                        return null;
+                    }
+                    fieldLimit = unionOctetsRO.limit();
+                }
+                break;
+            case FIELD_INDEX_FIELD1:
+                if ((bitmask & (1 << FIELD_INDEX_FIELD1)) != 0)
+                {
+                    optionalOffsets[FIELD_INDEX_FIELD1] = fieldLimit;
+                    fieldLimit += FIELD_SIZE_FIELD1;
+                }
+                break;
+            }
+        }
+        if (limit() > maxLimit)
+        {
+            return null;
+        }
+        return this;
+    }
+
+    @Override
+    public int limit()
+    {
+        return offset() + buffer().getByte(offset() + PHYSICAL_LENGTH_OFFSET);
+    }
+
+    @Override
+    public String toString()
+    {
+        final long bitmask = bitmask();
+        return String.format("LIST_WITH_UNION [bitmask=%s%s, field1=%d]",
+            String.format("0x%04X", bitmask),
+            (bitmask & (1 << FIELD_INDEX_UNION_OCTETS)) != 0 ? String.format(", unionOctets=%s", unionOctets()) : "",
+            field1());
```

Comment:
Effective bitmask includes defaulted fields, so ORing explicit bitmask with defaults mask gives effective bitmask for fields to be included in `toString()`.

Reply:
```
boolean unionOctetsIsSet = (bitmask & (1 << FIELD_INDEX_UNION_OCTETS)) != 0;

StringBuilder format = new StringBuilder();
format.append("LIST_WITH_UNION [bitmask=%s");
if (unionOctetsIsSet)
{
    format.append(", unionOctets=%s");
}
...
format.append("]");

return MessageFormat.format(format.toString(), unionOctetsIsSet ? unionOctets() : null, ...);
```